Kismet_CF_6.tex

```diff
@@ -0,0 +1,200 @@
+    \item The \secondaryfull{} is the backup for \primary{}, and communicates with OPS. If \primary{} becomes incapacitated, \secondary{} is responsible for removing them from danger.
+    \item The \daqfull{} monitors and operates the test data acquisition system.
+    \item the \controlfull{} operates the test control system, including actuation of remote valves.
+    \item The \heatfull{} operates the valves for the tank heating system.
+    \item \perifull{}, \periifull{}, and \periiifull{} ensure that no unauthorized personnel enter the testing area during test operations.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end personnel required section -- }}}
+
+% begin sign off -- {{{
+\subsection{Sign-Off}
+\textit{To be completed by all test personnel after reading and familiarization with procedures}
+\begin{checklist}
+    \item \opsfull      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \primaryfull  \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \secondaryfull\tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \daqfull      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \controlfull      \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \heatfull     \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \perifull     \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \periifull    \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+    \item \periiifull    \tabto{25em}\rule{10em}{0.4pt}\hspace{5em}\rule{10em}{0.4pt}
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end sign off -- }}}
+
+\newpage
+% begin prior to start section -- {{{
+\subsection{Prior to Start}
+\begin{checklist}
+    \item Ensure that the following procedures are complete:
+    \begin{checklist}
+        \item Oxidizer Tank Assembly procedure
+        \item Plumbing Setup procedure
+        \item Oxidizer Tank Stand Setup procedure
+        \item Tank Heating Setup procedure
+        \item Test Stand Setup procedure
+        \item Data Acquisition Setup procedure
+        \item Test Control System Setup procedure
+    \end{checklist}
+    \item Ensure that all technicians as defined above are available and have completed the sign-off.
+    \item Ensure that all spectators and test personnel are wearing safety glasses.
+    \item Ensure that \primary{} and \secondary{} are wearing face shields and have no exposed skin.
+    \item Ensure that \primary{} is wearing thermal gloves.
+    \item Ensure that \ops{} is in possession of the system control key.
+\end{checklist}
+\setcounter{checklistnum}{0}
+% end prior to start section -- }}}
+
+\newpage
+% begin Cold Flow Test Procedure section -- {{{
+\subsection{Cold Flow Test Procedure}
+\begin{checklist}
+    \item \ops{}: Confirm that the camera equipment is set up.
```

Comment:
also, move this to right after the pause point (item 30). We shouldn't start recording until the last possible minute, to ensure we maximize battery life